android-app/app/build.gradle

```diff
@@ -43,4 +43,5 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    implementation 'de.hdodenhof:circleimageview:3.1.0'
```

Comment:
Is this implementation somewhere used?

Reply:
Currently not, had only tried something with it